sysctl.d/50-default.conf

```diff
@@ -23,18 +23,27 @@ kernel.core_uses_pid = 1
 
 # Source route verification
 net.ipv4.conf.default.rp_filter = 2
-net.ipv4.conf.*.rp_filter = 2
--net.ipv4.conf.all.rp_filter
+# the below deviates from upstream systemd-245 (and later) since the default
```

Comment:
Can you add a noun between "the" and "below"?
e.g. "settings", "rules", "configurations", etc

Same in the other two comments.

Reply:
But... below *is* the [noun](https://www.merriam-webster.com/dictionary/below). I get your point wrt. being more specific though, will update.